regulations/static/components/CollapseButton.vue

```diff
@@ -1,17 +1,19 @@
 <template>
-    <div v-bind:class="{ visible: visible }">
-        <button v-on:click="click" aria-label="expand or collapse a subpart">
-            <slot name="expanded" v-if="visible">Hide</slot>
-            <slot name="collapsed" v-if="!visible">Show</slot>
-        </button>
-    </div>
+    <button
+        v-bind:class="{ visible: visible }"
+        v-on:click="click"
+        aria-label="expand or collapse a subpart"
+    >
+        <slot name="expanded" v-if="visible">Hide</slot>
+        <slot name="collapsed" v-if="!visible">Show</slot>
+    </button>
```

Comment:
removed containing div because it was being focused using keyboard navigation.  Button is also now full height and can be tapped/clicked anywhere in that height to toggle collapse

Reply:
Also added `data-test` data attribute here to better target elements for e2e tests; `data-test` is one of the options recommended in Cypress Best Practices here: https://docs.cypress.io/guides/references/best-practices#Selecting-Elements